eclipse-theia/theia-blueprint
# Update IDE to 1.46.0

#### What it does

* updates to Theia 1.46.0
* Updates builtins to 1.83.1
* fixes update documentation
* adds favicon for browser app

Fixes: #325

#### How to test

Build the application and check everything is still working

Regarding the favicon: This is working for me in Firefox, on Chrome it does not show up. I've checked the HTML and the entry is present and the link to the favicon is working (e.g. http://localhost:3000/favicon.ico), so I am not sure if this is some caching issue on my side. 
I will fix this in a follow up if required. 

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
